packages/use-optional-state/README.md

```diff
@@ -18,21 +18,21 @@ When implementing a component, it's sometimes hard to choose one or the other si
 
 This hook helps you to support both patterns in your components, increasing flexibility while also ensuring ease of use.
 
-Since the solution can be applied on a per-prop basis, you can even enable this behaviour for multiple props that are orthogonal (e.g. a `<Prompt isOpen inputValue="" />` component).
+Since the solution can be applied on a per-prop basis, you can also enable this behaviour for multiple props that are orthogonal (e.g. a `<Prompt isOpen inputValue="" />` component).
```

Comment:
It seems that the initial excitement has worn off 😉

Reply:
Haha, right 😆. I found it unnecessarily salesy on a second look 😁